homework/year_of_born.py

```diff
@@ -0,0 +1,2 @@
+age = int(input("enter your age: \n"))
+print("You were born in " + str(2018-age))
```

Comment:
good, but print(f"You were born in {2018-age}") looks better and shorter.
Also, it would be nice to set `2018` to the variable `year`